ocs_ci/ocs/ocp.py

```diff
@@ -289,30 +289,63 @@ def wait_for_resource(
                 False otherwise
 
         """
-        for sample in TimeoutSampler(
-            timeout, sleep, self.get, resource_name, True, selector
-        ):
+        log.info(
+            ("Waiting for a resource(s) of kind %s identified by name '%s'"
```

Comment:
consider using f strings

Reply:
Ok, it looks like we are always logging everything with DEBUG level anyway.